cram_pr2/cram_pr2_shopping_demo/resource/kitchen.urdf

```diff
@@ -981,12 +981,17 @@
 
  <!-- This is the vertical Drawer -->
  <!-- Joint oven area to the drawer -->
-  <joint name="oven_area_area_right_drawer_joint" type="fixed">
```

Comment:
I guess we don't need this kitchen anymore. We should just use the one from iai_maps. Please delete this file.

Reply:
4